webapp/app/templates/components.html

```diff
@@ -5,6 +5,14 @@
       <a class="{% if  float_right %} float-right {% endif %} btn btn-primary" href="{{ url }}">{{ text }}</a>
 {%- endmacro %}
 
+{% macro primaryButtonWithTracking(text, url, target, source, float_right=False) -%}
```

Comment:
Nice! like that! Can we add also a plausible to our current react anchor buttons if needed?

Reply:
Do you think we should add this with this PR or leave it for when we want to add more insights (we then also know which button we need to add the plausible to).

Reply:
Hmm for the future I think :)